static/README.md

```diff
@@ -66,15 +66,11 @@ Visit:
 
 ```go
 func (r *startCmd) Run() error {
-	dir, err := os.Getwd()
-	if err != nil {
-		log.Fatalf("could not get working directory: %v", err)
-	}
-	log.Printf("Process running from: %s", dir)
+	const publicPathPrefix = "/v1/public/"
 
 	cfg := svc.DefaultConfig()
 	// ...
-	cfg.SetPublicContentHandler(api.PublicContentHandlerOrPanic())
+	cfg.SetPublicContentHandler(api.PublicContentHandlerOrPanic(publicPathPrefix), publicPathPrefix)
```

Comment:
I'm getting a little lost -- is there ever a case where you don't pass the same string to both these calls?